src/pages/Editor/containers/editContext.js

```diff
@@ -11,10 +11,69 @@ const svgStyles = {
   top: 0,
   left: 0,
 };
-const EditContextProvider = props => {
+const EditContextProvider = (props) => {
   const aImagePrefix = "";
   const [pageSrc, setPageSrc] = useState(`${aImagePrefix}blank1.png`);
   const [isBody, setIsBody] = useState(true);
+  const [currentPage, setCurrentPage] = useState(0);
+  const [pages, setPages] = useState([
+    {
+      head: {
+        text: "",
+        headLeft: 0,
+        headTop: 0,
+        headWidth: null,
+      },
+      body: {
+        text: "",
+        bodyLeft: 0,
+        bodyTop: 0,
+        bodyWidth: null,
+      },
+    },
+  ]);
```

Comment:
```suggestion
  const initialState = [
    {
      head: {
        text: "",
        headLeft: 0,
        headTop: 0,
        headWidth: null,
      },
      body: {
        text: "",
        bodyLeft: 0,
        bodyTop: 0,
        bodyWidth: null,
      },
    },
  ]
  const [pages, setPages] = useState(initialState);
```
It's always a good practice to extract out the initial state declaration to a variable and make use of that variable to set the initial state.